CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.10.95](https://github.com/brave/brave-browser/releases/tag/v1.10.95)
+
+- Disabled Twitter support in the rewards panel. ([#10208](https://github.com/brave/brave-browser/issues/10208))
+- Fixed "Enable bottom toolbar" being reverted in certain cases. ([#9843](https://github.com/brave/brave-browser/issues/9843))
+- Fixed ads being displayed under YouTube in certain cases. ([#10241](https://github.com/brave/brave-browser/issues/10241))
```

Comment:
How about having it as "Fixed ads being shown on YouTube videos in certain cases"?

Reply:
I don't think we need "video" as it's basically implied that it's video content when mentioning YT. But used the above wording other than "video".